src/components/Books/Book.js

```diff
@@ -1,24 +1,31 @@
 import { CircularProgressbar } from 'react-circular-progressbar';
 import { PropTypes } from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { deleteBookThunk } from '../../redux/apiConnection';
+// import { deleteBook } from '../../redux/books/books';
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.